yellowbrick/features/pca.py

```diff
@@ -215,7 +226,7 @@ def finalize(self, **kwargs):
 
 def pca_decomposition(X, y=None, ax=None, features=None, scale=True,
                       proj_dim=2, proj_features=False, color=None,
-                      colormap=palettes.DEFAULT_SEQUENCE,
+                      legend_labels=None,colormap=palettes.DEFAULT_SEQUENCE,
```

Comment:
Again, let's change to `classes=None` here for consistency with other similar visualizers.